cmd/activator/main.go

```diff
@@ -295,13 +297,21 @@ func main() {
 	ah = activatorhandler.NewRequestEventHandler(reqChan, ah)
 	ah = tracing.HTTPSpanMiddleware("handle_request", ah)
 	ah = configStore.HTTPMiddleware(ah)
+	reqLogHandler, err := pkghttp.NewRequestLogHandler(ah, logging.NewSyncFileWriter(os.Stdout), "",
```

Comment:
Do we want some flag to turn on request logs like what we did for queue?

Reply:
If the template is empty, the feature is effectively turned off. The difference between this and queue proxy is that the value can change while activator is running and activator gets the notification and updates the handler. So, we need to inject the handler at all times and just re-evaluate the template str. With queue proxy, we only set this up at the startup and we can skip injecting the handler if template is empty (a good optimization).